pkg/users/group-details.js

```diff
@@ -0,0 +1,218 @@
+                                {_("Delete")}
+                            </Button>
+                        </CardActions>
+                    </CardHeader>
+                    <CardBody>
+                        <Form isHorizontal onSubmit={apply_modal_dialog}>
+                            <FormGroup fieldId="group-name" hasNoPaddingTop={!superuser.allowed} label={_("Name")}>
+                                { superuser.allowed
+                                    ? <TextInput id="group-name"
+                                                 value={edited_name !== null ? edited_name : group}
+                                                 onKeyPress={event => {
+                                                     if (event.key == "Enter") {
+                                                         event.target.blur();
+                                                     }
+                                                 }}
+                                                 onChange={value => set_edited_name(value)}
+                                                 onBlur={() => change_name()} />
+                                    : <output id="group-name">{group}</output>}
+                            </FormGroup>
+                            <GroupAccountsSelect accounts={accounts} group={group} groups={groups} />
+                        </Form>
+                    </CardBody>
+                </Card>
+            </PageSection>
+        </Page>
+    );
+}
+
+export const GroupAccountsSelect = ({ accounts, group, groups }) => {
+    const current_group = groups.find(g => g.name == group);
+    const [isOpenAccount, setIsOpenAccount] = useState(false);
+    const [selected, setSelected] = useState([...current_group.userlist, ...current_group.userlistPrimary]);
+    const [history, setHistory] = useState([]);
+    const [loggedInUsersAffected, setLoggedInUsersAffected] = useState(false);
+
+    const undoAccountChanges = () => {
+        const undoItem = history[history.length - 1];
+        if (undoItem.type === 'added') {
+            removeAccount(undoItem.name, true).then(() => setHistory(history.slice(0, -1)));
+        } else if (undoItem.type === 'removed') {
+            addAccount(undoItem.name, true).then(() => setHistory(history.slice(0, -1)));
+        }
+    };
+
+    const removeAccount = (account, isUndo) => {
+        if (!isUndo)
+            setHistory([...history, { type: 'removed', name: account }]);
+
+        const current_account = accounts.find(a => a.name == account);
+        if (current_account.loggedIn) setLoggedInUsersAffected(true);
+
+        return cockpit.spawn(["/usr/bin/gpasswd", "-d", account, group], { superuser: "require", err: "message" })
+                .then(() => {
+                    setSelected(selected.filter(item => item !== account));
+                    setIsOpenAccount(false);
+                }, error => {
+                    show_unexpected_error(error);
+                });
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230109-123701-1958283d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)